### PR TITLE
SorrTask489-Convert-Hubstaff-gdoc-to-Markdown

### PR DESCRIPTION
**Pull request by gurtajbir**
#489

### CODE REVIEW

**Comment by samarth9008** [comment]
Go through this [doc](https://github.com/sorrentum/sorrentum/blob/master/docs/First_review_process.md) and treat your PR accordingly.

**Review by DanilYachmenev — COMMENTED** [review]
good progress, see the comments

File: docs/Tools_Hubstaff.md
[the remark this answers is not in the document]
Comment (gurtajbir) [reply docs/Tools_Hubstaff.md:1]: Done

Comment (DanilYachmenev) [inline docs/Tools_Hubstaff.md:1]: no need in this doc rn, leave only 1 md file

File: docs/Tools_Hubstaff_figs/image1.png
Comment (DanilYachmenev) [inline docs/Tools_Hubstaff_figs/image1.png:1]: rename figs folder accordingly
-> `/Hubstaff_figs`

File: docs/Hubstaff.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+<!-- tocstop -->
+
+# General
+
+- [<span class="underline">Hubstaff</span>](https://hubstaff.com/) is a tool for
+remote working that automates:
+
+  - time tracking
+
+  -  invoice creation
```
Comment (DanilYachmenev) [inline docs/Hubstaff.md:21]: remove extra space between dash and the first word

**Review by DanilYachmenev — COMMENTED** [review]
couple nits and ready

do not forget to fix the conflict

File: docs/Hubstaff.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+with Hubstaff and get paid automatically for the worked hours
+
+# Time Tracking
+
+## Privacy
+
+- We have decided to turn off the feature of taking screenshots of the laptop as
+proof-as-work, URL, and app tracking
+
+  <img src="Hubstaff_figs/image4.jpg" style="width:6.26772in;height:4.66667in" />
```
Comment (DanilYachmenev) [inline docs/Hubstaff.md:36]: images are not being displayed, you can check yourself in the file preview

also I do not see the folder with `figs` which is supposed to contain the images
I suggest you to rerun the script on docx file to get the folder and working images

Comment (gurtajbir) [reply docs/Hubstaff.md:36]: Yes. there were some issues when I was trying to resolve the conflicting files. Will do this soon and update.

Comment (gurtajbir) [reply docs/Hubstaff.md:36]: Hi @DanilYachmenev. I was able to resolve all the above issues on a new copy of the repo on my end (the previous one got too messy). Would it be ok if I create a new PR?

**Pull request closed without merging**